skiko/src/jvmMain/cpp/windows/directXRedrawer.cc

```diff
@@ -19,18 +19,6 @@
 
 const int BuffersCount = 2;
 
-// This is a list of not supported graphics cards that have rendering issues (black screen, flickering)
-// with the current Swing/Skia integration.
-// If PC has other graphics cards suitable for DirectX12, one of them will be used. Otherwise,
-// rendering will falls back to OpenGL.
-const std::vector<std::wstring> notSupportedAdapters{
-    L"Intel(R) HD Graphics 520",
-    L"Intel(R) HD Graphics 530",
-    L"Intel(R) HD Graphics 4400",
-    L"NVIDIA GeForce GTX 750 Ti",
-    L"NVIDIA GeForce GTX 960M",
-    L"NVIDIA Quadro M2000M"};
```

Comment:
Are we skip this list of video adapters?
Or it moves to another place?

Reply:
It is [here](https://github.com/JetBrains/skiko/blob/master/skiko/src/jvmMain/resources/not-supported-adapter.list)

Reply:
Also, we have broken build on android